heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -151,6 +165,50 @@ function process_message()
     msg.Fields[11].value    = cnts[3] -- yahoo
     msg.Fields[12].value    = cnts[4] -- other
 
+    msg.Fields[13].value = ""
+    msg.Fields[14].value = ""
```

Comment:
Yeah, I debated that with the existing code but it was just cleaner and faster with the indexes (albeit less user friendly)